src/utils/isL2Chain.ts

```diff
@@ -0,0 +1,12 @@
+import { loadChains } from "../chains";
+import { Environment, FeeToken } from "../libs";
+
+export async function isL2Chain(
+  environment: Environment,
+  chain: string,
+  destinationNativeToken: FeeToken
+) {
+  const chains = await loadChains({ environment });
```

Comment:
do you think we need this? 
i.e. would `return destinationNativeToken.l1_gas_price_in_units` be enough? wondering if we can save another API call since i don't htink we need the `evmChains.includes(chain)` part

Reply:
yeah, i think that's enough